_kbarticles/2024-03-25-monitoring-app-metrics.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Monitoring your application's health"
+date: March 25, 2024
+excerpt: How to monitor the CPU and memory metrics for your application on cloud.gov
+
+---
+
+One of the benefits of hosting your applications on cloud.gov is that it offers [performance metrics for
+monitoring the health of your application](https://docs.cloudfoundry.org/loggregator/container-metrics.html).
+
+The most important metrics for monitoring your application's health are the memory and CPU metrics, which can help you identify:
+
+- If your application has enough memory provisioned per instance
+- If your application is experiencing CPU spikes and associated performance issues
+- If you need [to horizontally scale your application instances]({{ site.baseurl }}{% link _kbarticles/2023-08-07-advantages-of-multiple-application-instances.md %})
+
+## A note about CPU metrics
```

Comment:
I think this does a good job clarifying the difference between the metrics, but I also think the mentions about "scaling", and especially "scaling issues" could be kind of confusing - my first read through, I actually thought this was borrowed from docs on autoscaling